File: web3rpc/rpc-specs/code-samples/python/debug/others/storageRangeAt.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockHash = "0x90c81195698bc9f282bbdec386b0afb4dcc28e43aae834894281c3ecb3c88d21"
txIndex = 1
address = "0x73a7d19d14f7dfac5b799e405e22133b2adc57a6"
keyStart = "0x12"
maxResult = 1

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.storage_range_at(blockHash, txIndex, address, keyStart, maxResult)

print(debug_response)

File: web3rpc/rpc-specs/code-samples/javascript/debug/others/storageRangeAt.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");    
    const blockHash = "0x90c81195698bc9f282bbdec386b0afb4dcc28e43aae834894281c3ecb3c88d21";
    const txIndex = 1;
    const contractAddress = "0x73a7d19d14f7dfac5b799e405e22133b2adc57a6";
    const keyStart = "0x12";
    const maxResult = 1;

    sdk.debug.storageRangeAt(blockHash, txIndex, contractAddress, keyStart, maxResult, {}, (err, data, response) => {
        console.log(data);
    });
}
)()